File: tests/expected/sys_argv.dart
// @dart=3.4
import 'dart:io';
import 'package:sprintf/sprintf.dart';

main(List<String> argv) {
  List<String> a = (new List<String>.from([Platform.executable])..addAll(argv));
  String cmd = (a[0] ?? (throw Exception("key not found")));

  if (cmd == "dart") {
/* pass */
  } else {
    assert(cmd.contains("sys_argv"));
  }

  if (a.length > 1) {
    print(sprintf("%s", [(a[1] ?? (throw Exception("key not found")))]));
  } else {
    print(sprintf("%s", ["OK"]));
  }
}

File: tests/expected/go/sys_argv.go
package main

import (
	"fmt"
	"os"
	"strings"
)

func main() {
	var a []string = os.Args
	var cmd string = a[0]
	if cmd == "dart" {
		// pass

	} else {
		if !(strings.Contains(cmd, "sys_argv")) {
			panic("assert")
		}
	}
	if len(a) > 1 {
		fmt.Printf("%v\n", a[1])
	} else {
		fmt.Printf("%v\n", "OK")
	}
}
